packaging/language/maven_artifact.py

```diff
@@ -81,6 +80,12 @@
         required: true
         default: present
         choices: [present,absent]
+    validate_certs:
+        description: If C(no), SSL certificates will not be validated. This should only be used on personally controlled sites using self-signed certificates.
```

Comment:
Hmm... I think the documentation is unclear.  I meant that you should only change it from the default of validate-certs=yes if using a personally controlled repository.  I'l change it to be more clear.